src/docs/pages/form-validation.md

```diff
@@ -0,0 +1,226 @@
+
+- Integrating the Web Components with an associated `<form>` element and accessing their values using the `FormData` of the form
+- Directly accessing the props of a Web Component
+- Using the `detail` attribute of a custom event dispatched by the Web Component
+
+## Form association
+
+There are two ways of associating a form-related with an existing `<form>` element:
+
+```js
+// Using the form prop
+<form id="registrationForm" />
+<ld-input form="registrationForm" name="firstName" />
+
+// or wrapping it with a form element
+<form>
+  <ld-input name="firstName" />
+</form>
+```
+
+> To make the Web Component accessible via the `FormData` of the `<form>` element, the `name` prop of the component has to be set.
+
+If a form-related component is associated with a form in one of the above ways, the component's current value can be accessed using the form itself like it would be the case for a native `<input>` element.
+
+### Example
+
+{% example '{ "opened": true }' %}
+<form name="registrationForm1" style="display:flex;align-items:center">
+  <ld-label>
+    First Name*
+    <ld-input name="firstName" placeholder="Jane Doe" required></ld-input>
+    <ld-input-message style="visibility:hidden">This field is required.</ld-input-message>
+  </ld-label>
+  <ld-button style="margin:-0.3rem 0 0 1rem">Submit</ld-button>
+</form>
+
+<script>
+  const form = document.registrationForm1
+  const inputField1 = form.querySelector('ld-input')
+  const inputMessage1 = form.querySelector('ld-input-message')
+
+  function validateInput1() {
+    const formData = new FormData(form)
+
+    if (!formData.get('firstName')) {
+      inputField1.invalid = true
+      inputMessage1.style.visibility = 'inherit'
+      return false
+    }
+
+    inputField1.invalid = false
+    inputMessage1.style.visibility = 'hidden'
+    return true
+  }
+
+  inputField1.addEventListener('input', validateInput1)
+
+  inputField1.addEventListener('blur', validateInput1)
+
+  function getValues() {
+    const formData = new FormData(form)
+    let values = '';
+
+    formData.forEach((value, key) => {
+      values += `
+${key}: ${value}`
+    })
+
+    return values
+  }
+
+  form.addEventListener('submit', event => {
+    event.preventDefault()
+    const isValid = validateInput1()
+    const values = getValues()
+
+    // setTimeout is used in order to let the style update before showing the alert.
+    // You can probably dismiss the timeout, if you're using a UI framework like React or Vue.
+    setTimeout(() => {
+      if (isValid) {
+        window.alert(`Form submitted. FormData:${values}`)
+      } else {
+        window.alert('Form is invalid.')
+      }
+    }, 100)
+  })
+</script>
+{% endexample %}
+
+## Accessing props
+
+Accessing props works just as it would with native input elements. The props that a Liquid Oxygen component holds its content in can be found on the component's documentation page.
```

Comment:
```suggestion
Accessing props works just as it would with native input elements. The props, which allow you to access component specific values, are documented in each component's documentation page.
```